docs/docs/guides/build/assets/asset-selection-syntax.md

```diff
@@ -150,102 +132,116 @@ taxi_zones_file taxi_trips_file
+| **Two layers upstream of an asset** | `2+key:"my_asset"` | Selects assets two steps upstream from `my_asset`. |
+| **All layers downstream of an asset** | `key:"my_asset"+` | Selects all downstream assets that depend on `my_asset`. |
+| **One layer downstream of an asset** | `key:"my_asset"+1` | Selects assets that directly receive input from `my_asset`. |
+| **Two layers downstream of an asset** | `key:"my_asset"+2` | Selects assets two steps downstream from `my_asset`. |
+| **One layer upstream and downstream of an asset** | `1+key:"my_asset"+1` | Selects one layer of assets providing input to and receiving input from `my_asset`. |
+| **Two layers upstream and downstream of an asset** | `2+key:"my_asset"+2` | Selects two layers of assets providing input to and receiving input from `my_asset`. |
+| **All layers upstream and downstream of an asset** | `+key:"my_asset"+ ` | Selects all assets upstream and downstream of `my_asset`. |
+
+### Select all upstream layers \{#all-upstream}
+
+To select an asset and all its upstream dependencies, add a plus sign (`+`) before the asset key in the query.
 
-<Tabs>
+This example selects the `manhattan_map` asset and all its upstream dependencies.
+
+<Tabs groupId="examples">
 <TabItem value="python" label="Python">
 
 ```python
-taxi_zones_job = define_asset_job(name="taxi_zones_job", selection="*taxi_zones*")
+manhattan_all_upstream = AssetSelection.from_string("+manhattan_map")
 ```
 
 </TabItem>
 <TabItem value="cli" label="CLI">
 
-When selecting an asset's entire lineage using the CLI, enclose the asterisk (`*`) and the asset key in double quotes (`"`):
+When selecting an asset's dependencies using the CLI, enclose the plus sign (`+`) and the asset key in double quotes (`"`):
 
 ```shell
-dagster asset list --select "*taxi_zones*"
-dagster asset materialize --select "*taxi_zones*"
+dagster asset list --select "+manhattan_map"
+dagster asset materialize --select "+manhattan_map"
 ```
 
 </TabItem>
 <TabItem value="dagster-ui" label="Dagster UI">
 
 ```shell
-*taxi_zones*
++manhattan_map
 ```
 
 Which would result in the following asset graph:
 
-![Screenshot of Daggy U project graph](/images/guides/build/assets/asset-selection-syntax/select-entire-lineage.png)
+![TODO]()
 
 </TabItem>
 </Tabs>
 
----
-
-### Selecting upstream dependencies
-
-#### Selecting all upstream dependencies \{#all-upstream}
+### Select a specific number of upstream layers \{#specific-upstream}
 
-To select an asset and all its upstream dependencies, add an asterisk (`*`) before the asset key in the query.
+To select an asset and multiple upstream layers, add a number and plus sign (`+`) for the layers you want to select before the asset key in the query.
 
-This example selects the `manhattan_map` asset and all its upstream dependencies.
+This example selects the `manhattan_map` asset and two upstream layers.
```

Comment:
Hmm, this also seems redundant given the table above 🤔 . Maybe we move the table the bottom as a reference?